src/nile/core/account.py

```diff
@@ -84,7 +100,7 @@ def send(self, to, method, calldata, max_fee, nonce=None):
         )
 
         return call_or_invoke(
-            contract=self.address,
+            contract=self,
```

Comment:
This threw me off for a bit 😅

Reply:
I think is better to pass the account instance to avoid passing a lot of extra parameters like the ABI (that was required for predeployed accounts). All these params are in the instance already. Suggestions are welcome to address this, of course.

Reply:
Agreed on this being a better approach. Well done!

Reply:
this is causing a circular dependency, i'm rolling it back in #268 

```sh
____________________________________________________ ERROR collecting tests/commands/test_get_balance.py ____________________________________________________
ImportError while importing test module '/home/nile/tests/commands/test_get_balance.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/commands/test_get_balance.py:7: in <module>
    from nile.utils.get_balance import get_balance
.tox/default/lib/python3.8/site-packages/nile/utils/get_balance.py:3: in <module>
    from nile.core.call_or_invoke import call_or_invoke
.tox/default/lib/python3.8/site-packages/nile/core/call_or_invoke.py:6: in <module>
    from nile.core import account
.tox/default/lib/python3.8/site-packages/nile/core/account.py:20: in <module>
    from nile.core.call_or_invoke import call_or_invoke
E   ImportError: cannot import name 'call_or_invoke' from partially initialized module 'nile.core.call_or_invoke' (most likely due to a circular import) (/home/nile/.tox/default/lib/python3.8/site-packages/nile/core/call_or_invoke.py)
```